rtc/RobotHardware/robot.cpp

```diff
@@ -665,9 +665,11 @@ bool robot::checkJointCommands(const double *i_commands)
 bool robot::checkEmergency(emg_reason &o_reason, int &o_id)
 {
     int state;
+    joint_control_mode mode;
     for (unsigned int i=0; i<numJoints(); i++){
         read_servo_state(i, &state);
-        if (state == ON && m_servoErrorLimit[i] != 0){
+        read_control_mode(i, &mode);
+        if (state == ON && m_servoErrorLimit[i] != 0 && mode != JCM_TORQUE){
```

Comment:
In current JSK JAXON control board's imprementation, 
- `JCM_POSITION` means `position control`.
- `JCM_TORQUE` means `position control + command_torque`.

While the robot is hanged by the crane, we use  JCM_TORQUE with high position pdgain. After the robot is put on the ground, we reduce position pdgain (or set position pdgain to zero).

If  JCM_TORQUE with high position pdgain, servoErrorLimitCheck is necessary.
If JCM_TORQUE with low (or zero) position pdgain, servoErrorLimitCheck should be deactivated.

We currently modify `read_servo_state` function in our iob.cpp to switch servoErrorLimitCheck. 
```
int read_servo_state(int id, int *s)
{
  *s = 0;
  if (servo_state[id] == ON &&
      pgain[id] > THRESHOLD)
    *s = 1;
  return TRUE;
}
```

After this PR is merged, JCM_TORQUE mode always disables servoErrorLimitCheck. So we have to change our software. @kindsenior